src/main/scala/ch/epfl/bluebrain/nexus/iam/realms/Realms.scala

```diff
@@ -1,8 +1,108 @@
 package ch.epfl.bluebrain.nexus.iam.realms
 
-import ch.epfl.bluebrain.nexus.iam.types.{AuthToken, Caller}
+import java.time.Instant
+import java.util.concurrent.TimeUnit
 
-class Realms[F[_]] {
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import cats.Monad
+import cats.effect._
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient
+import ch.epfl.bluebrain.nexus.iam.acls.Acls
+import ch.epfl.bluebrain.nexus.iam.config.AppConfig.{HttpConfig, RealmsConfig}
+import ch.epfl.bluebrain.nexus.iam.index.KeyValueStore
+import ch.epfl.bluebrain.nexus.iam.realms.RealmCommand.{CreateRealm, DeprecateRealm, UpdateRealm}
+import ch.epfl.bluebrain.nexus.iam.realms.RealmEvent.{RealmCreated, RealmDeprecated, RealmUpdated}
+import ch.epfl.bluebrain.nexus.iam.realms.RealmRejection._
+import ch.epfl.bluebrain.nexus.iam.realms.RealmState.{Active, Current, Deprecated, Initial}
+import ch.epfl.bluebrain.nexus.iam.realms.Realms.next
+import ch.epfl.bluebrain.nexus.iam.types.IamError.{AccessDenied, UnexpectedInitialState}
+import ch.epfl.bluebrain.nexus.iam.types._
+import ch.epfl.bluebrain.nexus.rdf.Iri.{Path, Url}
+import ch.epfl.bluebrain.nexus.sourcing.akka.AkkaAggregate
+import io.circe.Json
+
+/**
+  * Realms API.
+  *
+  * @param agg   the realms aggregate
+  * @param acls  a lazy acls api
+  * @param index an index implementation for realms
+  * @param http  the application http configurations
+  * @tparam F    the effect type
+  */
+class Realms[F[_]: MonadThrowable](agg: Agg[F], acls: Lazy[F, Acls], index: RealmIndex[F])(implicit http: HttpConfig) {
+
+  private val F = implicitly[MonadThrowable[F]]
```

Comment:
Then why not add `F: MonadThrowable[F]` to the implicit parameters?

Reply:
to avoid line wrapping :)